cmp/grid/columns/Column.js

```diff
@@ -651,7 +652,7 @@ export class Column {
                     tooltipSpec(val, {record, column: this, gridModel, agParams}) :
                     val;
 
-                return ret ?? null;
+                return isValidElement(ret) ? ret : toString(ret);
```

Comment:
Don't we still have to explicitly return null here rather than an empty string if ret is null/undefined? 

My understanding (haven't tested) is that this will display an empty tooltip, rather than no tooltip.